consensus/src/alerts/mod.rs

```diff
@@ -398,13 +406,19 @@ impl<'a, H: Hasher, D: Data, MK: MultiKeychain> Alerter<'a, H, D, MK> {
     }
 }
 
+pub(crate) type MessagesForNetwork<H, D, MK> = Sender<(
+    AlertMessage<H, D, <MK as Keychain>::Signature, <MK as MultiKeychain>::PartialMultisignature>,
```

Comment:
Would it be enough to alias the inner type, instead of the whole `Sender`? It would be a bit cleaner. Same for `Receiver`.

Reply:
Sure, done.

Reply:
No problem.